scaleway/resource_iot_hub.go

```diff
@@ -189,6 +201,8 @@ func resourceScalewayIotHubCreate(ctx context.Context, d *schema.ResourceData, m
 			return diag.FromErr(err)
 		}
 	}
+	MQTTUrl := computeIotHubCaURL(req.ProductPlan)
```

Comment:
Output an error or warn on error

Reply:
Output an error if mqtt_ca is empty and warn if there is already a certificate stored

Reply:
```suggestion
return diag.Diagnostics{{
  Severity: diag.Warning,
  Path: cty.AttrPath("mqtt_ca")
}}
```